pgd_splicer/ProcessPDBTask.py

```diff
@@ -405,8 +405,48 @@ def parseWithBioPython(path, props, chains_filter=None):
     # Be kind; rewind.
     decompressed.seek(0)
 
+    alt_structure = Bio.PDB.PDBParser().get_structure('pdbname',
+                                                  decompressed.name)
+    io=PDBIO()
+    io.set_structure(alt_structure)
+    io.save("not-ordered.pdb")
+
+
+    for ichain in alt_structure[0] :
+        for ires in ichain :
+            if ires.is_disordered():
+                res_sum = {}
+                count_sum = {}
+                for atom in ires :
+                    if atom.is_disordered():
+                        try :
+                            res_sum[ atom.get_altloc() ] = res_sum[atom.get_altloc()] + atom.get_occupancy()
+                        except KeyError :
+                            res_sum[atom.get_altloc()] = atom.get_occupancy()
+                        try :
+                            count_sum[atom.get_altloc()] = count_sum[atom.get_altloc()]+1
+                        except KeyError :
+                            count_sum[atom.get_altloc()] = 1
+
+                avg_dict = {}
+                for k,v in res_sum.iteritems() :
+                    avg_dict[k] = res_sum[k]/count_sum[k]
+
+                keepAltID = max(k for k, v in avg_dict.iteritems())
+                for atom in ires :
+                    if ( not atom.is_disordered()) or atom.get_altloc()  ==  keepAltID :
+                        atom.set_altloc(' ') 
+
+    io=PDBIO()
+    io.set_structure(alt_structure)
+    io.save("ordered.pdb")
+    #I'm currently not using this pdb file for further use in this file
+    #Even though the test pass, i need confirmation from you
```

Comment:
You should definitely use this PDB file in the output.  We need to see what happens!